src/utils/opensearch_init.py

```
@@ -0,0 +1,212 @@
+        logger.info(
+            "Created OpenSearch index for traditional connector service",
+            index_name=index_name,
+            vector_dimensions=INDEX_BODY["mappings"]["properties"]["chunk_embedding"][
+                "dimension"
+            ],
+        )
+        await TelemetryClient.send_event(
+            Category.OPENSEARCH_INDEX, MessageId.ORB_OS_INDEX_CREATED
+        )
+
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)